### PR TITLE
cr#12 and cr#13

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

File: src/Classes/GameController.h [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+            move::MoveController * moveController;
+            std::vector<IGameEventListener *> listeners;
+        public:
+            static cocos2d::Scene* createScene();
+            virtual bool init();
+            CREATE_FUNC(GameController);
+            void registerListener(IGameEventListener * listener);
+        };
+    }
+}
```
Comment (stavenko) [inline src/Classes/GameController.h]: Could we use some kind of auto-formatting. Like http://clang.llvm.org/docs/ClangFormat.html

File: src/Classes/GameController.cpp [hunk 2]
```diff
@@ -22,119 +24,117 @@ Scene* GameController::createScene()
...
-//    auto batch = SpriteBatchNode::create("data/characters/knight/IDLE/0.png");
-//    auto sprite = Sprite::createWithTexture(batch->getTexture(), Rect(0,0,64,147));
-//    sprite->setPosition(center);
-//    map->addChild(sprite);
-
+    //todo refactor below code
+    //
     //
+    //
+//    Size visibleSize = Director::getInstance()->getVisibleSize();
```
Comment (stavenko) [inline src/Classes/GameController.cpp]: I think, it's nice habit to avoid commented code in production.

Comment (apocarteres) [inline src/Classes/GameController.cpp]: agree. will drop it.

File: src/Classes/GameController.h (hunk 1 above)
Comment (apocarteres) [inline src/Classes/GameController.h]: nice catch. i'll take a look

**Comment by apocarteres** [comment]
solves https://github.com/TopSecretGames/game/issues/13

**Comment by apocarteres** [comment]
solves #12

**Pull request merged**